docs/content/latest/api/ysql/commands/cmd_do.md

```diff
@@ -0,0 +1,73 @@
+---
+title: DO
+linkTitle: DO
+summary: DO
+description: DO
+menu:
+  latest:
+    identifier: api-ysql-commands-do
+    parent: api-ysql-commands
+aliases:
+  - /latest/api/ysql/commands/cmd_do
```

Comment:
nit: I don't think you need the alias here. That's only for backwards compatibility with old/changed links.
But please double-check that -- if unsure it's probably safe to leave it.